imports/ui_3/views/components/comments-section.html

```diff
@@ -22,14 +26,21 @@
 <template name="Comment">
     {{#unless this.isHiddenBy currentUser._id }}
     <div class="social-comment">         
-        <a href="{{pathFor 'User.show' _id=user._id}}" class="pull-left">
-            <img alt="avatar" class="message-avatar" src={{user.avatar}}>
+        <a href="{{pathFor 'User.show' _id=this.user._id}}" class="pull-left">
+            <img alt="avatar" class="message-avatar" src={{this.user.avatar}}>
         </a>
         <div class="media-body" >
-            <a href="{{pathFor 'User.show' _id=user._id}}">
-                {{user}}
+            <a href="{{pathFor 'User.show' _id=this.user._id}}">
+                {{this.user}}
             </a>
-            <span class="newlines" data-id="{{_id}}">{{>Chopped text=text}}</span>
+            <span {{b "unless: editing"}} class="newlines" data-id="{{_id}}">{{>Chopped text=text}}</span>
```

Comment:
Ez a {{b "unless: editing"}} kulonbozik attol mintha egy {{#unless editing}}-be lenne a span?

Reply:
Mert ha nem, akkor picit olvashatobb, ha az alatta levo #if-nek - ha mar ugyis ott van- az else agaba tesszuk.

Reply:
Teljesen ugyanaz, csak így rövidebbnek tűnt, de tényleg érthetőbb az else ág, legyen úgy.

Reply:
Amugy ez erdekes szintaxis, tenyleg tomorebb mint a blaze-es #if es #unless.
Nem is tudtam hogy a viewmodel ennyire ki tudja valtani a blaze dolgait is.
Potencialisan akkor lehetne a kodot ilyen {{b-kel irni {{#if-ek helyett...